### PR TITLE
Draft minutes for Package WG Meeting 29/5/19

### PR DESCRIPTION
**Pull request by drbenmorgan**
Copied as is from Live Notes apart from grammar/spelling corrections. @gartung and @chissg may also want to check I've accurately reflected their updates!

### CODE REVIEW

File: organization/_posts/2019/2019-05-29-packaging.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - Print "spec tree" for packages specified for development
+  - Input DAG (everything need for a release) can have several top-level packages with co-concretization
+  - Package checkout improvements, e.g. require that development packages come from VCS systems.
+  - Global build can handle tests in parallel
+- MVP1a
+  - Integration testing round that identified previously mentioned updates
+  - LArSoft now building with Art 3.02.04 (but will go to 3.02.05 for MVP)
+  - Remaining issue with "wirecell" package, need to support Git submodules
+    - Spack supports submodules, but wirecell needs a additional manual step.
+    - Guilherme: Could create tarballs? Chris: Would imply FNAL becomes maintainer of those tarballs. Fix locally in Spack to get MVP progressing
```
Comment (chissg) [inline organization/_posts/2019/2019-05-29-packaging.md]: Extra note: Brett VIren (wirecell maintainer) has already produced several composite source tarballs, the URIs for which have been incorporated into the recipe. A successful build of `0.10.9` has been achieved.

File: organization/_posts/2019/2019-05-29-packaging.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Chris: some issues still require updates to the core concretization system, aim is end of US fiscal year, but may slip.
```
Comment (chissg) [inline organization/_posts/2019/2019-05-29-packaging.md]: Note: these updates to the concretization system are coming from Spack founding author (Todd Gamblin).

File: organization/_posts/2019/2019-05-29-packaging.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+layout: meetings
+---
+
+# {{page.title}}
+
+## Agenda
+[<span class="underline">https://indico.cern.ch/event/819329/</span>](https://indico.cern.ch/event/819329/)
+
+Participants: Ben Morgan, Patrick Gartung, Graeme Stewart, Javier Cervantes, Chris Burr,
+Pere Mato, Dimitri, Geri Ganis, Guilherme Amadio, Marco Clemencic, Chris Green, Elizabeth Sexton-Kennedy
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-05-29-packaging.md]: To add "Ivan Razumov"
And Dimitri is "Dmitri Konstantinov"

I had to look up some people's names!

File: organization/_posts/2019/2019-05-29-packaging.md (hunk 2 above)
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-05-29-packaging.md]: I think "US fiscal year" is too obscure for non-US people. I suggest "end of September".

Comment (graeme-a-stewart) [reply organization/_posts/2019/2019-05-29-packaging.md]: Indeed, that's probably worth adding.

File: organization/_posts/2019/2019-05-29-packaging.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- Chris: some issues still require updates to the core concretization system, aim is end of US fiscal year, but may slip.
+
+## Distributing CMS FWLite with Conda (Patrick)
+- Conda recap:
+  - Latest uses gcc 7.3/clang 4.0.1 (with SDK from OSX 10.9)
+  - Everything linked under one prefix
+- FWLite with scram in Conda?
+  - Conda prefix + ROOT installed
+  - Linux: Trivial
+  - OSX: issues with Scram going into infinite (with/without SIP)
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-05-29-packaging.md]: infinite loop?

File: organization/_posts/2019/2019-05-29-packaging.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+- Went to CMake using scram2cmake tool
+  - See https://github.com/gartung/fwlite
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-05-29-packaging.md]: Could make that a link (just add <>s)

**Pull request merged**